src/pages/AddPersonalBankAccountPage.js

```diff
@@ -92,17 +106,15 @@ class AddPersonalBankAccountPage extends React.Component {
             >
                 <HeaderWithBackButton
                     title={this.props.translate('bankAccount.addBankAccount')}
-                    onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_PAYMENTS)}
+                    onBackButtonPress={this.exitFlow}
                 />
                 {shouldShowSuccess ? (
                     <ConfirmationPage
                         heading={this.props.translate('addPersonalBankAccountPage.successTitle')}
                         description={this.props.translate('addPersonalBankAccountPage.successMessage')}
                         shouldShowButton
                         buttonText={this.props.translate('common.continue')}
-                        onButtonPress={() => {
-                            Navigation.navigate(ROUTES.SETTINGS_PAYMENTS);
-                        }}
+                        onButtonPress={this.exitFlow}
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/38558, this has caused a small issue where the same page was added twice to the navigation stack